src/dialog/overflow-menu/overflow-menu-option.component.ts

```diff
@@ -78,8 +89,32 @@ export class OverflowMenuOption implements AfterViewInit {
 	 * disable/enable interactions
 	 */
 	@Input() disabled = false;
-
+	/**
+	 * If it's an anchor, this is its location
+	 */
 	@Input() href: string;
+	/**
+	 * Allows to add a target to the anchor
+	 */
+	@Input() set target(value: string) {
```

Comment:
Looks good, though I'd keep the type as well as the `TARGETS` map ... Perhaps an `enum` would make sense here?

Reply:
Tried but with no success... Enums have no methods unless `Object.keys(enum)` or `Object.entries(enum)` and are [string, string] tuples for each value, so it's kind of a bigger overhead IMHO.
But will go that road if you think it's best.

Reply:
You know what, it's done. Let me know what y'all think.